INSTALL.md

```diff
@@ -13,7 +13,25 @@
 
 # Prequisites
 
-Since currently CLI runs all of it's tasks in [Docker](https://www.docker.com/), make sure it's [installed](https://docs.docker.com/get-docker/).
+Since currently CLI runs all of it's tasks either in [Docker](https://www.docker.com/) or [Podman](https://podman.io/), make sure one of these is installed.
+
+## Docker
+
+Docker is available on a variety operating systems and distributions.
+
+OS-specific instructions can be found here: https://docs.docker.com/get-docker/
+
+## Podman
+
+Podman is generally only available on Linux.
+
+Distribution-specific instructions can be found here: https://podman.io/getting-started/installation
```

Comment:
Probably it's worth to add a note about how to switch backends if both Docker and Podman are installed on the host.

Reply:
Documented in https://github.com/cirruslabs/cirrus-cli/pull/189/commits/9bceefc8bdf6f69e898c0907559dd370adc1e7dd.